### PR TITLE
Update deprecated modules and use absolute paths for revealjs paths

### PR DESCRIPTION
**Pull request by mimosinnet**
As discussed in [Mojolicious Forums](https://groups.google.com/forum/#!topic/mojolicious/22PKGB4gMjU), as the plugin uses relative paths it does not work on nested routes. This pull request adds absolute paths in revealjs template and updates deprecated modules.

### CODE REVIEW

File: lib/Mojolicious/Plugin/RevealJS.pm [hunk 1]
```diff
@@ -41,7 +41,7 @@ sub _include_code {
     % require Mojo::Util;
     % my $file = stash 'revealjs.private.file';
     <pre><code class="<%= stash('language') // 'perl' %>" data-trim>
-      <%= Mojo::Util::slurp(app->home->rel_file($file)) =%>
+      <%= Mojo::File::slurp(app->home->rel_file($file)) =%>
```
Comment (jberger) [inline lib/Mojolicious/Plugin/RevealJS.pm]: Mojo::File::slurp is not a function, it is a method. `app->home->rel_file($file)->slurp`

**Comment by jberger** [comment]
I used your commits (squashed) for absolute paths (a4472f073f30e1713d1669ed3c112a51f1fd276f) I've written my own patch for the deprecated function usage (5c374306b250d710563c278322300efdc4ab81c6).

Thanks

**Comment by jberger** [comment]
I have real reservations about this fix now (for the absolute paths). I host lots of my talks on github pages and therefore the static resources must be served from a relative (or non /) location. As I'm presenting at TPC next week, some kind of change will have to be made by then. I'll try not to break you @mimosinnet but this have become very inconvenient for me :s

**Comment by mimosinnet** [comment]
Ups. I am sorry! Feel free to change it at your convenience. The issue is that the plugin does not work in a Mojolicious full app (only works in a Lite app) when using relative paths. Knowing the issue I can manage it. Thanks!

**Comment by jberger** [comment]
Perhaps this could be used? http://mojolicious.org/perldoc/Mojolicious/Guides/Cookbook#Rewriting

**Comment by jberger** [comment]
Actually, I'm going to open a new issue for this rather than reusing this one

**Pull request closed without merging**